values.yaml

```diff
@@ -640,7 +640,7 @@ meshGateway:
     additionalSpec: null
 
   # Envoy image to use.
```

Comment:
Do we need a note saying if you're running Consul version x you need envoy xxx?

Reply:
Consul 1.7+ needs envoy 1.13+? Or 1.11+?